engine/src/juliabox/handlers/handler_base.py

```diff
@@ -399,8 +399,18 @@ def redirect_to_logged_in_instance(self, user_id):
         self.log_info("Logged in %s", "nowhere" if loggedin_instance is None else "here already")
         return False
 
-    def post_auth_launch_container(self, user_id):
+    def post_auth_launch_container(self, user_id, skip_post_auth_plugins=[]):
+        for plugin in JBPluginHandler.jbox_get_plugins(JBPluginHandler.JBP_HANDLER_POST_AUTH):
+            if plugin in skip_post_auth_plugins:
```

Comment:
Can't `skip_post_auth_plugins` be avoided by not configuring the skipped plugins to be loaded in the first place?

If the purpose is to keep it dynamic, that's probably something the plugins should do themselves. Having the caller specify the skip list would necessarily need the caller to be aware of the post auth plugins configured, which is not nice. Right?